### ISSUE TITLE
add extra checks in "VM test extras" for handover

### ISSUE THREAD

**Issue by arif-ali** [canonical/ua-reviewkit#34]
Currently we have the following "VM test extras" (if applicable)

* GPU passthrough flavor
* 2 x VMs with DPDK flavor and test east-west traffic
* 2 x VMs with SRIOV flavor and test east-west traffic

I think we should add further network tests

* 2 x VMs with ovs/ovn vxlan and test east-west traffic
* 2 x VMs with ovs/ovn flat provider network and test east-west traffic
* 2 x VMs with ovs/ovn vlan provider network and test east-west traffic

I know this will give the FEs more things to test, but this will ensure we catch any issues early

**Comment by dosaboy** [comment]
Agreed, if we can get this into tempest (perhaps including the existing checks) that would be preferable.

**Comment by dosaboy** [comment]
@nobuto-m to what extend does/could magpie cover this and if it did would that be helpful?

**Comment by nobuto-m** [comment]
If I'm not mistaken, Juju doesn't support multiple network spaces on top of OpenStack, so it would be cumbersome to test both tunneled traffic (VXLAN, Geneve) and provider network traffic.

On top of that, one of the benefits of Tempest (or existing extra checks) was that it didn't require the archive / internet access as a must. In some customer cases, it's impossible to run Juju agents at all because 1. tunneled traffic may not have a gateway to the internet, 2. tenant / provider networks are for more restricted traffic policy. Tempest has been okay for such a restricted environment so far.
